homework -15

Tootienahs opened the pull request.

Tootienahs committed:
First commit on my fork

fork a cloned repo from testPro A59

Tootienahs committed:
second commit on my fork

forked a clone repo from A59..

Tootienahs committed:
Merge branch 'main' of https://github.com/Tootienahs/A59-QA-Automation-Framework

Tootienahs committed:
Updated the URL

Updated the URL to testpro.io

Tootienahs committed:
Merge pull request #1 from Tootienahs/git-ntro

Updated the URL

Berichicko commented:
Hello,

I have checked your homework 15, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

![804deb64dc406a20a75da05d91a1a69e](https://user-images.githubusercontent.com/104761482/235961479-9afb2841-b657-46c9-99f8-2402f933b843.jpg)

![image](https://github.com/Test-Pro-Bootcamp/A59-QA-Automation-Framework/assets/104761482/b5f82fa6-2e76-4704-9fdf-6b534127f524)

Tootienahs committed:
homework 17

I got stuck on xpath

Tootienahs committed:
homework -17-

home work late but it toke longer to get. I did not finish yet. but please check.

Tootienahs committed:
homework update 17

Tootienahs committed:
new homework17

Tootienahs committed:
homework 17-- new

Tootienahs committed:
Merge pull request #3 from Tootienahs/homework-17-

Homework 17

Berichicko commented:
Hello,

I have checked your homework 15, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

![Thumbs-up-hand-with-crown-illustration-on-transparent-background-PNG](https://github.com/Tootienahs/A59-QA-Automation-Framework/assets/104761482/20861dce-f8e7-4435-9b92-de6436933d0b)

![Quote](https://github-readme-quotes-bay.vercel.app/quote?theme=dark&layout=socrates)


![image](https://github.com/Test-Pro-Bootcamp/A59-QA-Automation-Framework/assets/104761482/ff1367e1-5ca6-4379-9d53-b46ffdce0429)
